doc/source/data_modules.rst

```diff
@@ -3,164 +3,157 @@
-merged without overwriting keys from inherited dictionaries, except where this
-is wanted. Using the same namespace in a more specific dictionary (i.e. a lower
-level directory) can be used to change or even remove that namespace (for the
-latter set it to `Null`).
-
-`kiwi_bootloader_param` refers to any bootloader type parameter supported by
-`kiwi <https://documentation.suse.com/kiwi/9/html/kiwi/building-types.html#disk-bootloader>`__.
-
-`kiwi_preferences_type_param` refers to any preferences type parameter supported
-by `kiwi` (see `\<preferences\>\<type\> in kiwi documentation
-<https://documentation.suse.com/kiwi/9/html/kiwi/image-description.html#sec-preferences>`__).
-
-`kernelcmdline` is not a string that is direct copied into the appropriate
-`kiwi` parameter but a dictionary that defines kernel parameters individually,
-with each key representing a kernel parameter. This allows to inherit parts of
-the kernel command line from other modules. There are two notations for
-parameters.  `kernel_param: value` will be translated into a single
-`kernel_param=value`, and `kernel_multi_param: [value, value, ...]` will add
-`kernel_multi_param` multiple times for each value from the given list.
-
-`packages_type` can be `bootstrap` or `image` (see `kiwi documentation
-<https://documentation.suse.com/kiwi/9/html/kiwi/image-description.html#sec-packages>`__).
-The items in the package list have two possible notations. Either just a plain
-string, which is considered to be the package name, or a dictionary with keys
-`name` (the package name) and `arch` (the build architecture for which the
-package should be included).
-
-List items in `config` `script` refer to files in :file:`data/scripts` (with
-:file:`.sh` appended by `keg`) and the content of those will be added to the
-:file:`config.sh` script.
-
-List items in `config` `services` refer to system service that should be
-enabled or disabled in the image. Analogue to packages, there are two supported
-version, a short one containing only the service name, or a long one that
-allows to specify whether the service should be enabled or disabled).
-
-`setup` has the same structure as `config` but the data will be used to
-generated :file:`images.sh` instead of :file:`config.sh`.
-
-List items in `overlayfiles` refer to directories under
-:file:`data/overlayfiles`. Files from those directories will be copied into
-an overlay archive to be included in the image, either a generic or a profile
-specific one (depending on where the data module was included), or a named one
-in case `archivename` tag is used.
+    _namespace_base_services:
+      services:
+        - sshd
+
+
+Merging based on the parent key allows for grouping for different types of
+configuration data in one data module.
+
+
+Image Configuration Scriptlets
+------------------------------
+
+Configuration scriptlets are individual script snippets that can be used
+to generate image configurations script. `KIWI` runs those scripts at
```

Comment:
"...generate image configurations script" "...the generate image configurations script" ? But then again a kiwi config can have 2 scripts, config.sh and images.sh. We currently do not handle images.sh. As such this can be clarified I think.

Reply:
I fixed the typo ("configurations script" -> "configuration scripts"). Keg can produce `images.sh`. A kiwi config can have more than two scripts actually (see https://documentation.suse.com/kiwi/9/single-html/kiwi/index.html#working-with-kiwi-user-defined-scripts), but currently only `config.sh` and `images.sh` are supported by keg.